static/src/javascripts/lib/config.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import pad from 'lib/pad';
+// eslint-disable-next-line guardian-frontend/global-config
+const config = window.guardian.config;
+
+export default Object.assign(
+    {
+        hasTone(name) {
+            return (this.page.tones || '').indexOf(name) > -1;
```

Comment:
this would done automatically by `fix` if it wasn't an object property, so good case for it